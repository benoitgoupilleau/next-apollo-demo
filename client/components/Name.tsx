type NameProps = {
  name: string;
};

const Name = ({ name }: NameProps) => {
  return <span>{name}</span>;
};

export default Name;
