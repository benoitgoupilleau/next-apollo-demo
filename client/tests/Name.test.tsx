import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Name from '../components/Name';

describe('Name component', () => {
  it('should render component correctly', () => {
    const { container, getByText } = render(<Name name="hello" />);

    expect(container).toMatchSnapshot();
    expect(getByText('hello')).toBeInTheDocument();
  });
});
