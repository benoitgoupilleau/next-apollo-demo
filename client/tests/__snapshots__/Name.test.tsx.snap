// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name component should render component correctly 1`] = `
<div>
  <span>
    hello
  </span>
</div>
`;
